eclipse/plugin/src/nl/tudelft/watchdog/eclipse/ui/wizards/ProjectRegistrationInputPanel.java

```diff
@@ -124,8 +125,8 @@ private boolean registerProject() {
 		}
 
 		Preferences preferences = Preferences.getInstance();
-		preferences.registerProjectId(project.name, projectId);
-		preferences.registerProjectUse(project.name, true);
+		preferences.registerProjectId(WatchDogUtils.getWorkspaceName(), projectId);
```

Comment:
Aha, this is the bug we uncovered last time!

Reply:
That was actually a different bug, but yes they were closely related to each other :smile: